app/controllers/email/VerifyEmailController.scala

```diff
@@ -58,27 +58,7 @@ class VerifyEmailController @Inject()(val emailVerificationService: EmailVerific
 
   def emailSendVerification: Action[AnyContent] = blockAgentPredicate.async { implicit user =>
 
-    if (appConfig.features.emailPinVerificationEnabled()) {
-      Future.successful(Redirect(routes.VerifyPasscodeController.emailSendVerification()))
-    } else {
-      implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(user.headers, Some(user.session))
-
-      extractSessionEmail match {
-        case Some(email) =>
-          emailVerificationService.createEmailVerificationRequest(email, routes.ConfirmEmailController.updateEmailAddress().url).map {
-            case Some(true) => Redirect(routes.VerifyEmailController.emailShow())
-            case Some(false) =>
-              logDebug(
-                "[VerifyEmailController][sendVerification] - " +
-                  "Unable to send email verification request. Service responded with 'already verified'"
-              )
-              Redirect(routes.ConfirmEmailController.updateEmailAddress())
-            case _ => errorHandler.showInternalServerError
-          }
-
-        case _ => Future.successful(Redirect(routes.CaptureEmailController.show()))
-      }
-    }
+    Future.successful(Redirect(routes.VerifyPasscodeController.emailSendVerification()))
```

Comment:
There are also some additional things you can remove that are now no longer needed. Could you remove the following routes and their associated controller actions and tests:

- /verify-email-address
- /contact-preference/verify-email-address
- /contact-preference/send-verification

Off the back of that could you also remove the view `VerifyEmailView`, the English and Welsh messages that were used only in that view, and the unit tests for the view.